src/main/java/ether/JSON.java

```diff
@@ -0,0 +1,147 @@
+package ether;
+
+import io.left.rightmesh.id.MeshID;
+import io.left.rightmesh.mesh.MeshManager;
+import io.left.rightmesh.util.EtherUtility;
+import io.left.rightmesh.util.MeshUtility;
+import io.left.rightmesh.util.RightMeshException;
+import org.json.simple.JSONObject;
+
+import java.math.BigInteger;
+
+
+/**
+ * The JSON wrapper
+ */
+public final class JSON {
```

Comment:
Is this  JSON wrapper? It appears to be sending messages on the mesh. A more descriptive name might help.

Reply:
Wrapping the creation of JSON objects into a class might not be a bad idea, but that logic shouldn't have a "sending over the network" side effect.